.github/dependabot.yaml

```diff
@@ -0,0 +1,62 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    labels:
+      - dependencies
+      - team/agent-platform
+      - changelog/no-changelog
+      - qa/skip-qa
+    milestone: 22
+    ignore:
```

Comment:
If the ignore list here is how the deps in `replace` in `go.mod` are ignored, could you mention this in a comment in the `go.mod` file (so that we remember to update the dependabot config when we add/remove `replace` directives)

Reply:
Have we reached consensus with relevant teams regarding the ignore list? I'll try to make sure we can vet this list. Let me know if you already have @mx-psi!

Reply:
> If the ignore list here is how the deps in replace in go.mod are ignored, could you mention this in a comment in the go.mod file (so that we remember to update the dependabot config when we add/remove replace directives)

Added comments on c18dcf82619301e3cf24444b01dbdb188a33f124 to clarify the reason they are added.

> Have we reached consensus with relevant teams regarding the ignore list? I'll try to make sure we can vet this list. Let me know if you already have @mx-psi!

No, I am waiting to have a review from each team

Reply:
@olivielpeau @truthbk most teams had a look at the list now, could you give it another look so we have an approval from Agent Core on this PR?